index.js

```diff
@@ -217,6 +217,73 @@ var MapToNamespace = function(entity, collection) {
   db[collection].push(entity);
 };
 
+var RemoveFromNamespace = function(db, table, collection) {
```

Comment:
Yep, but I can remove it for now since it is not sufficiently tested and also not in use yet. 👍